packages/web-components/fast-foundation/docs/guide/styling-components.doc.md

```diff
@@ -3,4 +3,91 @@ id: styling-components
 title: Styling Components
 sidebar_label: Styling Components
 custom_edit_url: https://github.com/microsoft/fast-dna/edit/master/packages/web-components/fast-foundation/docs/guide/styling-components.doc.md
----
\ No newline at end of file
+---
+## Applying CSS
+`@microsoft/fast-components` leverages `@microsoft/fast-element` to style components. For details on how to apply CSS, see [leveraging CSS](fast-element/leveraging-css.md).
+
+## Design System
+In FAST, the *design system* is a collection of properties and values that inform the visual design language of the components. These properties are managed and provided through implementations of the *DesignSystemProvider*. See the [*DesignSystemProvider* documentation](fast-foundation/fast-design-system-provider.md) for more information.
+
+For the purposes of this section, the *DesignSystemProvider* is a Custom Element that will create CSS Custom Properties that can be consumed by component stylesheets and provide mechanisms to synchronize, update, and consume those properties programmatically.
+
+### Type Ramp
+The FAST type ramp is exposed by the `FASTDesignSystemProvider` as CSS Custom Properties. It centers the ramp around a _base_ font size and line-height, ascending and descending from the _base_. The CSS Custom Properties that can be used are:
```

Comment:
Could "centers" be replaced with "organizes" or some other word.  I initially interpreted this as a styling type of center.